### ISSUE TITLE
ffmpeg version N-93185-g6fc762b Copyright (c) 2000-2019 the FFmpeg developers

### ISSUE THREAD

**Issue by jnorthrup** [acoustid/ffmpeg-build#1]
tl;dr -- what forces the ffmpeg binary to be built when clipping filters and parsers out?

i noticed a few wrinkles in present-day.

when i disable things en-mass, the most common headache is that ffmpeg binary is skipped.

the script in this repo dates back to ffserver.  that interrupts configure, but i like having this reference. 

what i have tried with inconsistent results on different ubuntu versions is to build a default binary then go back and *try* to kill some video specific compilations.

with e.g.
 
--disable-encoder={$(ffmpeg -encoders|grep "^ [SV]"|cut -f 3 -d' ' |grep -e '\w\+'  | xargs echo |tr ' ' ,)} \
 --disable-decoder={$(ffmpeg -decoders|grep "^ [SV]"|cut -f 3 -d' ' |grep  -e '\w\+'|   xargs echo |tr ' ' ,)}\
  --disable-filter={$(ffmpeg -filters|grep  -e "\->V"|cut -f 3 -d' ' |  xargs echo |tr ' ' ,)}

with the filters being cut out in this way, the ffmpeg binary is disabled.  

is there a pivotal set of exe minimums that must be re-included after mass disables?

**Comment by lalinsky** [comment]
I'm sorry, but I don't understand what are you asking.

**Comment by kingkingkingson** [comment]
the version "N-93185-g6fc762b" how could I know from github commit or tags?
